code generators/de.fhdo.lemma.model_processing.code_generation.container_base/pom.xml

```diff
@@ -0,0 +1,106 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>de.fhdo.lemma.model_processing.code_generation.docker</groupId>
+    <artifactId>de.fhdo.lemma.model_processing.code_generation.docker</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Docker Code Generator</name>
+    <description>A LEMMA code generator for dockerfiles.</description>
+
+    <properties>
+        <java.version>11</java.version>
+        <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
+        <lemma.version>0.7.0-SNAPSHOT</lemma.version>
```

Comment:
When you specified the version as suggested above, you may remove this property and use ${project.version} instead.